### PR TITLE
Add cleanup for gcp cli mirroring

### PR DESCRIPTION
**Pull request by AHarmlessPyro**

### CODE REVIEW

File: cli/src/gcp/cleanup.ts [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+        }]
+    )
+    let packetMirrorName = identifySourceResp["_packetMirrorName"]
+    let mirrorInstance = mirroring.find((mirror) => mirror.name == packetMirrorName)
+    let regexPattern = /metlo-packet-mirroring-([a-z0-9\-]*)/gm;
+    let metloRegexResp = regexPattern.exec(mirrorInstance.name)
+    if (metloRegexResp.length != 2) {
+        throw new Error("Packet Mirroring policy didn't match expected pattern. Please try manual deletion.")
+    }
+    let metloId = regexPattern.exec(mirrorInstance.name)[1]
```
Comment (akshay288) [inline cli/src/gcp/cleanup.ts:40]: validate that this is a uuid

File: cli/src/gcp/cleanup.ts [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+
+    spinner.start("Deleting packet mirroring")
+    // Delete GCP Packet Mirroring
+    try {
+        let resp_pack = await conn.stop_packet_mirroring({
+            packetMirroringURL: mirrorInstance.selfLink.split("/").at(-1),
```
Comment (akshay288) [inline cli/src/gcp/cleanup.ts:46]: pull out and validate uuid

File: cli/src/gcp/cleanup.ts [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+        spinner.stop()
+        console.warn(err)
+        throw new Error(
+            `Couldn't delete GCP Forwarding Rule ${mirrorInstance.collectorIlb.url}`,
+        )
+    }
+    spinner.succeed("Deleted Forwarding Rule")
+
+    spinner.start("Deleting Load Balancer")
+    const [backends, ,] = await conn.list_backend_service()
```
Comment (akshay288) [inline cli/src/gcp/cleanup.ts:75]: check whole name

File: cli/src/gcp/cleanup.ts [hunk 4]
```diff
@@ -0,0 +1,249 @@
...
+        console.warn(err)
+        throw new Error(
+            `Couldn't delete GCP Backend Service ${backend.name}`,
+        )
+    }
+    spinner.succeed("Deleted Load Balancer")
+
+    spinner.start("Deleting Health Check")
+    const [healthChecks, ,] = await conn.list_health_checks()
+    const check = healthChecks.find((_check) => _check.name.includes(metloId))
```
Comment (akshay288) [inline cli/src/gcp/cleanup.ts:94]: check whole name

**Review by akshay288 — APPROVED** [review]

File: cli/src/gcp/cleanup.ts (hunk 2 above)
Comment (AHarmlessPyro) [reply cli/src/gcp/cleanup.ts:47]: done

**Pull request merged**